src/main/java/com/example/java/test/junior/developer/security/ApplicationPropertySourceFactory.java

```diff
@@ -0,0 +1,25 @@
+package com.example.java.test.junior.developer.security;
+
+import java.io.IOException;
+import java.util.Properties;
+import org.springframework.core.env.PropertiesPropertySource;
+import org.springframework.core.env.PropertySource;
+import org.springframework.core.io.Resource;
+import org.springframework.core.io.support.DefaultPropertySourceFactory;
+import org.springframework.core.io.support.EncodedResource;
+import org.springframework.core.io.support.PropertiesLoaderUtils;
+
+public class ApplicationPropertySourceFactory extends DefaultPropertySourceFactory {
```

Comment:
Why do you need custom property source factory? You don't use any custom properties source files or types.